cloud/gcp/storage/main.tf

```diff
@@ -0,0 +1,75 @@
+resource "google_kms_key_ring" "keyring" {
+  name     = "civform-keyring"
+  location = var.region
+}
+
+resource "google_kms_crypto_key" "storage-key" {
+  name            = "cloud-storage-key"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "604800s" //7 days
```

Comment:
question (blocking): what is the consequence of the rotation period for developers using this code?

Reply:
question (non-blocking): I thought the developers won't be able to use cloud storage anyway. restricted to ECS?

Anyway, a developer accessing the object in this bucket will need reader permissions on the customer managed key.

Reply:
Ah, I meant a developer deploying CiviForm using this code. What significance does the rotation period have?

Reply:
I don't think it has any significance on deployment. You just need to setup correct IAM bindings for the keys.